### PR TITLE
Updating Metaphlan tool: 

### PR DESCRIPTION
**Pull request by EngyNasr**
- adding the updated new version number and citation in macros.xml

- replacing metagonmics with microbiota in metaphlan.xml and removing a ls in a command 

- solving missing (assert_contents) tag error in customize_metaphlan _database.xml

### CODE REVIEW

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
ping @bebatut 

Thanks @EngyNasr and welcome to IUC :)

File: tools/metaphlan/metaphlan.xml [hunk 1]
```diff
@@ -211,7 +210,7 @@ python '$__tool_directory__/formatoutput.py'
         <section name="inputs" title="Inputs" expanded="true">
             <conditional name="in">
                 <param name="selector" type="select" label="Input(s)">
-                    <option value="raw" selected="true">Fasta/FastQ file(s) with metagenomic reads</option>
+                    <option value="raw" selected="true">Fasta/FastQ file(s) with microbiota reads</option>
```
Comment (bebatut) [inline tools/metaphlan/metaphlan.xml:213]: Could you also change in the param object with name `selector` that is below to keep a consistency?

**Review by bebatut — COMMENTED** [review]
I just put a small comment but looks good otherwise! Thanks @EngyNasr

File: tools/metaphlan/metaphlan.xml (hunk 1 above)
Comment (EngyNasr) [reply tools/metaphlan/metaphlan.xml:213]: I will add it now, thanks to you for all your guidance :)

File: tools/metaphlan/test-data/no_taxon_input.fasta [hunk 2]
```diff
@@ -0,0 +1,5 @@
+> seq1
+ATTAGGGATTTTAGGGGGGGAGATTTAGAGAGAGAGAGAGAGAAGAAGAGAAGAAGAAGAAGAAAAAGGGGGAAGAGAGA
+> seq2
+ATTAGGGATTTTAGGGGGGGAGATTTAGAGAGAGAGAGAGAGAAGAAGAGAAGAAGAAGAAGAAAAAGGGGGAAGAGAGA> seq1
```
Comment (bgruening) [inline tools/metaphlan/test-data/no_taxon_input.fasta:4]: this looks wrong, is that the intention?

Comment (EngyNasr) [reply tools/metaphlan/test-data/no_taxon_input.fasta:4]: So, yes, this should result in an UNKNOWN, which caused an error in the output files before when we run the tool. But now by skipping the resulting UNKNOWN the error is gone.  So this fasta file is added today in the test data folder in order to have a new test in the tool’s xml file for this case

Comment (bebatut) [reply tools/metaphlan/test-data/no_taxon_input.fasta:4]: I think what Björn wanted to say if that the  `> seq1` in last line should be removed

Comment (EngyNasr) [reply tools/metaphlan/test-data/no_taxon_input.fasta:4]: Okayyyy, sorry .. I will comit it in a few :))

Comment (bgruening) [reply tools/metaphlan/test-data/no_taxon_input.fasta:4]: I assume a line break is missing. But I also wonder how can this pass the test, this is not a valid FASTA file. Maybe Metaphlan is not that strict?

Comment (bebatut) [reply tools/metaphlan/test-data/no_taxon_input.fasta:4]: I am also wondering now...

File: tools/metaphlan/metaphlan.xml [hunk 3]
```diff
@@ -362,6 +361,152 @@ python '$__tool_directory__/formatoutput.py'
...
     </outputs>
     <tests>
+        <test expect_num_outputs="6">
+            <section name="inputs">
+                <conditional name="in">
+                    <param name="selector" value="raw"/>
+                    <conditional name="raw_in">
+                        <!-- Single GZ file -->
+                        <param name="selector" value="single"/>
+                        <param name="in" value="no_taxon_input.fasta.gz"/>
```
Comment (bebatut) [inline tools/metaphlan/metaphlan.xml:371]: Could you use the fasta file here directly and remove the fasta.gz file ?

Comment (EngyNasr) [reply tools/metaphlan/metaphlan.xml:371]: Yes sure :)

**Comment by bebatut** [comment]
Thanks @EngyNasr ! Congrats for your first PR on IUC !

**Pull request merged**